payjoin-cli/src/app.rs

```diff
@@ -477,4 +487,6 @@ impl payjoin::receive::Headers for Headers<'_> {
     }
 }
 
-fn serialize_psbt(psbt: &Psbt) -> String { base64::encode(&psbt.serialize()) }
+fn serialize_psbt(psbt: &Psbt) -> String {
```

Comment:
in general I'd like to see formatting in a separate commit to semantic changes.

This crate is formatted with nightly, `cargo +nightly fmt`, which this PR appears to fail.